src/screens/FeedScreen/Header.js

```diff
@@ -139,6 +139,7 @@ const _renderAnonimity = ({
                 karmaScore={karmaScore}
                 anonUserInfo={anonUserInfo}
                 version={version}
+                withKarma
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withKarma` prop has been added to the component, but it doesn't seem to have a value assigned to it. If it's meant to be a boolean prop that is always true, this is fine. However, if it's supposed to take a variable or function, you should assign a value to it.
```diff
- withKarma
+ withKarma={value}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->